src/instruments/src/PFD/shared/PFDSimvarPublisher.tsx

```diff
@@ -0,0 +1,358 @@
+import { EventBus, SimVarDefinition, SimVarValueType, SimVarPublisher } from 'msfssdk';
+
+export interface PFDSimvars {
```

Comment:
I assume these should match PFDVars 1:1? Some of the casing is slightly different.

There's also a way to enforce this with Typescript, but I'm struggling to write the precise type right now (you'd need a separate type with the boolean keys or something).

Reply:
I'll have a look (and at the very least unify the casing) 👍

Reply:
Casing should be the same now, unfortunately couldn't find the type to enforce it